rbs/rbs_common.cc

```diff
@@ -34,4 +34,21 @@ core::LocOffsets RBSDeclaration::typeLocFromRange(const rbs_range_t &range) cons
     }
     return comments.front().typeLoc;
 }
+
+bool isVisibilitySend(const parser::Send *send) {
+    return send->receiver == nullptr && send->args.size() == 1 &&
+           (parser::isa_node<parser::DefMethod>(send->args[0].get()) ||
+            parser::isa_node<parser::DefS>(send->args[0].get())) &&
+           (send->method == core::Names::private_() || send->method == core::Names::protected_() ||
+            send->method == core::Names::public_() || send->method == core::Names::privateClassMethod() ||
+            send->method == core::Names::publicClassMethod() || send->method == core::Names::packagePrivate() ||
+            send->method == core::Names::packagePrivateClassMethod());
+}
+
+bool isAttrAccessorSend(const parser::Send *send) {
```

Comment:
Same, let's move it to https://github.com/sorbet/sorbet/blob/master/parser/helper.h#L365